org.rdkit.knime.types/rdkit-chemsrc/org/rdkit/knime/EnvironmentChecker.java

```diff
@@ -3,7 +3,7 @@
  * This source code, its documentation and all appendant files
  * are protected by copyright law. All rights reserved.
  *
- * Copyright (C) 2014
```

Comment:
When you update a file, you should leave the original copyright in, so please use: 2014-2017